### ISSUE TITLE
[openweathermap] OpenWeatherMap Account offline until Weather and Forecast thing created

### ISSUE THREAD

**Issue by Moxified** [openhab/openhab-addons#11191]
## Expected Behavior
OpenWeatherMap Account thing should show as Online after creation even without dependent things.

## Current Behavior
After properly configuring an OpenWeatherMap Account thing, the resultant thing stays offline until you create a new dependent Weather and forecast (or possibly other OWM thing).

## Steps to Reproduce (for Bugs)
1. Have no OWM things.
2. Create new OpenWeatherMap Account thing
3. do not have any dependent things

## Context
While not a show stopper, it burned a significant amount of time debugging and attempting to get the "Account" thing to come online thinking it was configured improperly.

## Your Environment
* Version used: OH 3.1
* Environment name and version (e.g. Chrome 76, Java 8, Node.js 12.9, ...): Chromium Edge Version 92.0.902.84 (Official build) (64-bit)
* Java: OpenJDK 64-Bit Server VM Zulu11.50+19-CA (build 11.0.12+7-LTS, mixed mode)
* Operating System and version (desktop or mobile, Windows 10, Raspbian Buster, ...): Debian 11 VM w/o desktop environment

**Comment by lolodomo** [comment]
Looking at the code, I can confirm what you noticed. 
The way the bridge thing status is set, depending on its things status,, is relatively unusual but this is not really a bug. The bridge status is set to ONLINE only when one of its things is able to retrieve some data. 
Maybe it should be set to UNKNOWN rather than OFFLINE when there are no child things.

**Comment by Moxified** [comment]
I'm fairly new to the OH3 (and OH2 paperUI) way of doing things so I don't have any real basis to make claims.  What I can say is that this plugin behaved "unusually" when compared to the other bindings I had already configured and I spent a significant amount of time troubleshooting my config, OWM account and firewall settings before stumbling across the fact that it just behaves differently.

All that to say, something, either documentation, a flashing light or other would be appreciated to indicate the account thing is valid and to continue forth with OWM forecast (or similar) dependent things.

Feel free to close the issue if it is working as expected and no action is to be taken.  Thanks! for your hard work!
